cli/parser/parser.go

```diff
@@ -353,11 +371,17 @@ func getCommandLineSchema(args []string, bazelHelp BazelHelpFunc) (*CommandLineS
 	return schema, nil
 }
 
+func CanonicalizeStartupArgs(args []string) ([]string, error) {
+	return canonicalizeArgs(args, func(topic string) (string, error) {
+		return defaultStartupArgs, nil
+	}, true)
```

Comment:
would it work to pass `onlyStartupOptions` to `getCommandLineSchema` and early-return after the `schema` is initially created and populated with the parsed startup options? Then we can let `runBazelHelpWithCache` do its thing without needing to hard-code the startup_options help

Reply:
Good idea, I thought about doing something like this - but I didn't realize it was this simple. Done.